Ports/dosfstools/package.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='dosfstools'
+version='4.2'
+useconfigure='true'
+use_fresh_config_sub='true'
+config_sub_paths=('config.sub')
+files="https://github.com/dosfstools/dosfstools/archive/refs/tags/v${version}.zip dosfstools-${version}.zip bed48396285facb98e1d323a523654977fb5ddf350597bf4b4de11ae667fd585"
+auth_type='sha256'
+configopts=("--prefix=${DESTDIR}/usr" "--oldincludedir=${DESTDIR}/usr/include/")
```

Comment:
Why does this appear to install into `/usr`, but we later try to symlink to binaries in `/usr/local`?

Reply:
It was not needed actually, so I will remove this.